### DISCUSSION TITLE
Starting a Codespace - UX discrepancy

### DISCUSSION THREAD

**Discussion by bflint68** [community/community#55798] [Codespaces]
### Select Topic Area

Bug

### Body

When one starts a **new** codespace, another tab is opened (ie you retain a tab with the repo and you get another tab with the codespace)
If you open an existing codespace, it opens in the same tab.

My preference is for the former in both cases, ultimately, it should be consistent.

**Comment by samruddhikhandale** [comment]
Hi 👋 

Thanks for reporting, we appreciate it. 
I have opened an internal issue for tracking this discrepancy.

**Reply by bflint68** [reply]
thank you!

i **love** codespaces.
